basic_samples/DataViews/Python3/program.py

```diff
@@ -70,67 +80,88 @@ def createData(ocsClient):
         description="This is a sample Sds type for storing Pressure type "
                     "events for DataViews",
         sdsTypeCode=SdsTypeCode.Object,
-        properties=[pressureDoubleProperty, timeDateTimeProperty])
-    temperature_SDSType = SdsType(
-        id=sampleTemperatureTypeId,
-        description="This is a sample Sds type for storing Temperature type "
+        properties=[pressureDoubleProperty, temperatureDoubleProperty, timeDateTimeProperty])
+
+    pressure_SDSType2 = SdsType(
```

Comment:
In the original sample there were separate types and streams for Pressure and Temperature. With these changes that is no longer true (both types have both pressure and temperature), should we consider removing "pressure" from the type and stream variable names? E.G. use `SDSType1` `SDSType2`, `Stream1` `Stream2`, `Values1` `Values2`, `Val1` `Val2`, etc throughout the file.